### PR TITLE
Update FifaSoccerBall.proto

### PR DESCRIPTION
**Pull request by omichel**
Following https://github.com/cyberbotics/webots/pull/2782#pullrequestreview-600861979, this improves the FifaSoccerBall proto.

### CODE REVIEW

File: projects/objects/balls/protos/FifaSoccerBall.proto [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
-  field SFRotation rotation    0 1 0 0
-  field SFString   name        "fifa soccer ball"
-  field SFInt32    size        5 # Official FIFA ball sizes ranging from 1 to 5.
+  field SFVec3f            translation 0 0 0
+  field SFRotation         rotation    0 1 0 0
+  field SFString           name        "fifa soccer ball"
+  field SFInt32{1,2,3,4,5} size        5 # Defines the ball size according to FIFA standards.
 ]
 {
   %{
```
Comment (DavidMansolino) [inline projects/objects/balls/protos/FifaSoccerBall.proto:15]: Thank you.
You can remove this lua part as with this you are sure the value is in the correct range:
```
    if size < 1 then
      size = 1
      io.stderr:write("'size' must be strictly positive\n")
    elseif size > 5 then
      size = 5
      io.stderr:write("'size' must be less than or equal to 5\n")
    end
```

File: docs/guide/object-balls.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ FifaSoccerBall {
 
 ### FifaSoccerBall Field Summary
 
-- `size`: official FIFA ball sizes ranging from 1 to 5.
+- `size`: Defines the ball size according to FIFA standards. This field accepts the following values: `1`, `2`, `3`, `4` and `5`.
```
Comment (lukicdarkoo) [inline docs/guide/object-balls.md:79]:
```suggestion
- `size`: Defines the ball size according to FIFA standards. This field accepts the following values: `1`, `2`, `3`, `4`, and `5`.
```

> an Oxford comma to separate the last item in a list of three or more things.

https://prowritingaid.com/grammar/1008093/Is-it-grammatically-correct-to-put-a-comma-before-and-#:~:text=It%20is%20grammatically%20correct%20to,it%20as%20an%20Oxford%20comma.

The Oxford comma is used in American English only:
https://getproofed.com/writing-tips/differences-american-british-punctuation/

**Review by lukicdarkoo — APPROVED** [review]
Nice, thank you

**Review by DavidMansolino — APPROVED** [review]
Thank you.

**Pull request merged**